common/src/main/java/com/wynntils/utils/SystemUtils.java

```diff
@@ -47,13 +47,7 @@ public static int getMemMax() {
         return (int) (Runtime.getRuntime().maxMemory() / (1024 * 1024));
     }
 
-    private static final class ClipboardImage implements Transferable {
-        private final Image image;
-
-        private ClipboardImage(Image image) {
-            this.image = image;
-        }
-
+    private record ClipboardImage(Image image) implements Transferable {
```

Comment:
Same here. This is not a record. You can't just apply the IntelliJ suggestions without thinking.